src/app/_translate/lang-en.ts

```diff
@@ -88,6 +88,7 @@ export const LANG_EN_TRANS = {
   loadmore: 'Load more',
   log: 'Log',
   mapping: 'Mapping',
+  'max-results-reached': "You've reached the limit of 200 displayed executions",
```

Comment:
Perhaps, until we have implemented the new ticket regarding parametrized translation and so on, we could just remove all references to the number here? So perhaps:
"You've reached the limit of the number of executions that can be displayed."
The idea is that we are still going to experiment with the precise amount, and I don't think we should have something hard-coded.

Reply:
Done